qiskit_braket_provider/providers/adapter.py

```diff
@@ -405,7 +405,6 @@ def _simulator_target(description: str, properties: GateModelSimulatorDeviceCapa
 def _qpu_target(description: str, properties: DeviceCapabilities):
```

Comment:
With these additions, this method is becoming overly complex; please split it into multiple methods